examples/gui/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.html
```

Comment:
don't we have a .gitignore at the root level? You should update that

Reply:
No need for this since last few commits so I removed it